README.md

```diff
@@ -2,31 +2,283 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Demonstrate the problem of code duplication
+- Demonstrate a solution using inheritance
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Another object-oriented programming pillar is inheritance, which is the
+ability for one class to inherit from another class. We can think
+of inheritance as a hierarchical classification where a class
+can inherit another class' features (fields and methods).
+In this lesson, we will explore how to use inheritance in Java.
 
-## Topic 1
+## The Problem with Code Duplication
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Before we implement an application with inheritance, we'll see an
+example that does not use inheritance and instead relies
+on code duplication.  Code duplication can result in maintenance issues
+when the code needs to change.
+
+Let's assume our application needs to keep inventory of the items stocked in a grocery store.
+Every grocery item has an id, name, and price.  Some grocery items are perishable,
+so they have an expiration date in addition to the id, name, and price.
+
+Our initial model is shown in the UML
+class diagram below.   `PerishableItem` has the same fields
+and methods as `GroceryItem`, along with an additional field `expiration`
+and methods `getExpiration()` and `setExpiration()` (highlighted in yellow).
+Everything not highlighted in yellow represents duplication between the two classes.
+
+![class diagram version1](https://curriculum-content.s3.amazonaws.com/6677/pillars/classdiagram1.png)
+
+The Java implementation for `GroceryItem` and `PerishableItem` are shown
+below.  Notice it takes some effort in looking at the code to figure out
```

Comment:
I know this is probably the point, but maybe to help students out a little when looking at the code below, add a comment next to `public String expiration;` to give them a little hint?

Reply:
ok added comment next to field and methods